### PR TITLE
Implement Sodg::kids

### PR DESCRIPTION
**Pull request by UARTman**
Resolves https://github.com/objectionary/sodg/issues/6 and https://github.com/objectionary/sodg/issues/2.

### CODE REVIEW

File: src/ops.rs [hunk 1]
```diff
@@ -125,7 +125,11 @@ impl Sodg {
 
     /// Find all kids of a vertex.
     pub fn kids(&self, _v: u32) -> Result<Vec<(String, u32)>> {
-        Err(anyhow!("Not implemented yet"))
+        let vert = self
+            .vertices
+            .get(&_v)
+            .context(format!("Can't find v{_v}"))?;
```
Comment (yegor256) [inline src/ops.rs:131]: @UARTman see which "v" we use in other methods, it's not English `v`

File: src/ops.rs [hunk 2]
```diff
@@ -125,7 +125,11 @@ impl Sodg {
 
     /// Find all kids of a vertex.
     pub fn kids(&self, _v: u32) -> Result<Vec<(String, u32)>> {
```
Comment (yegor256) [inline src/ops.rs:127]: @UARTman I think it's better to rename it to `v`

File: src/ops.rs [hunk 3]
```diff
@@ -125,7 +125,11 @@ impl Sodg {
...
-        Err(anyhow!("Not implemented yet"))
+        let vert = self
```
Comment (yegor256) [inline src/ops.rs:128]: @UARTman we usually call it `vtx`

**Comment by yegor256** [comment]
@UARTman see some comments above

**Comment by UARTman** [comment]
@yegor256 Done.

**Comment by yegor256** [comment]
@rultor merge

**Comment by UARTman** [comment]
@yegor256 I had to fix formatting, but it's probably ready to merge now.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**